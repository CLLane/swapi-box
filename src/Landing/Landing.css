

.crawler-section {
  display: flex;
  justify-content: center;
  height: 1200px;
  color: #feda4a;
  font-family: 'Pathway Gothic One', sans-serif;
  font-size: 500%;
  font-weight: 600;
  letter-spacing: 6px;
  line-height: 150%;
  text-align: justify;
  overflow: hidden;
  perspective: 400px;
}

.crawler-div {
  position: relative;
  top: -100px;
  transform-origin: 50% 100%;
  animation: crawl 45s linear;
  animation-iteration-count: infinite;
}


.crawl-text {
  /* position: relative;
  top: -100px;
  transform-origin: 50% 100%;
  animation: crawl 45s linear;
  animation-iteration-count: infinite; */
}

@keyframes crawl {
  0% {
    top: 0;
    transform: rotateX(20deg) translateZ(0);
  }

  100% {
    top: -6000px;
    transform: rotateX(25deg) translateZ(-2500px);
  }
}
