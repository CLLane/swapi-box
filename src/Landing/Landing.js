import React from 'react';
import PropTypes from 'prop-types';
import './Landing.css'

const Landing = ({film}) => {
  console.log('film :', typeof film);
  return (
    <section className='crawler-section'>
      <div className='crawler-div'>
        <p>{film}</p>
      </div>
    </section>
  );
}



export default Landing;

Landing.propTypes = {
  film: PropTypes.string,
}