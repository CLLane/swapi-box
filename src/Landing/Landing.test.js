import React from 'react';
import { shallow } from 'enzyme';
import Landing from './Landing';

describe('Landing', () => {
  it('should match the snapshot with all data passed in correctly', () => {
    const film = '';
    const wrapper = shallow(<Landing
      film={film}
      />)
    expect(wrapper).toMatchSnapshot()
  })
})