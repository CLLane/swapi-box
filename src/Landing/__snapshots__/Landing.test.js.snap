// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing should match the snapshot with all data passed in correctly 1`] = `
<section
  className="crawler-section"
>
  <div
    className="crawler-div"
  >
    <p />
  </div>
</section>
`;
